### ISSUE TITLE
mention / -> ctrl-f trick in help

### ISSUE THREAD

**Issue by bovine3dom** [tridactyl/tridactyl#4860]
If you open the quick search bar you can then edit the settings by pressing ctrl+f and changing them there

Probably best on new tab

**Comment by sschalhorn-ionos** [comment]
Isn't that just launching the normal Firefox search bar, which is bound to ctrl+f even without Tridactyl?

**Comment by bovine3dom** [comment]
Yeah, but Tridactyl overrides ctrl+f on pages but not inside the search box. It's a Firefox feature that becomes more useful if you're using Tridactyl
